Makefile

```diff
@@ -1712,8 +1719,8 @@ rustup-install-target-toolchain: rustup-set-version
 # BASE_BRANCH and BASE_TAG will be automatically determined if not specified.
```

Comment:
Is this comment still true? Based on the usage, I assume not (usually you would not provide the flag if the value is empty).

Reply:
It is true! I updated it so it is more obvious but `--base-branch=""` is equivalent to no flag in this case and same for base-tag. Not sure if we it's something we want to fail but it matches behavior of the current script.